lib/scope.js

```diff
@@ -184,7 +185,7 @@ function defineScope(cls, targetClass, name, params, methods, options) {
   // are same
   if (cls === targetClass) {
     cls._scopeMeta[name] = params;
-  } else if (targetClass) {
+  }  else if (targetClass) {
```

Comment:
Two spaces after `}`, there should be only one. Just revert this line back please.